content/02-hello-world/02-first-app.md

```diff
@@ -12,18 +12,16 @@ import Html exposing (div, text)
 
 
 main =
-    div []
-        [ text "Hello"
-        ]
+    div [] [ text "Hello" ]
 ```
 
-In the terminal run:
+Back at the terminal prompt, run:
 
 ```
 elm-reactor
 ```
 
-Then open http://localhost:8000 and click on `Hello.elm`. This should show `Hello`. Excellent, our first Elm app!
+Then open http://localhost:8000 and click on `Hello.elm`. This should display the text “Hello” on an otherwise blank page. Excellent—we have our first Elm app!
```

Comment:
I find this style of not having a space around the `-` strange. Is that a common convention?